src/Compiler/ComponentTagCompiler.php

```diff
@@ -236,4 +241,27 @@ protected function parseAttributeBag(string $attributeString): string
 
         return (string) preg_replace($pattern, ' :attributes="$1"', $attributeString);
     }
+
+    /**
+     * Remove Twig comments from given attribute string
+     */
+    protected function pourOutCommentsFromAttributeBag(string $attributeString): string
+    {
+        $pattern = "/
+            (                                               # First capturing group
+                \{\#                                        # Match {# literally
+                    (                                       # Second capturing group
+                        [                                   # Match any character in the following list
+                            \w                              # Match words
+                            \d                              # Match digits
+                            \s                              # Match whitespace characters
+                            $&+,:;=?@#|\"'`<>.^*()%!-       # Match special characters
```

Comment:
```suggestion
                            $&+,:;=?@#|\"'`<>.^*()%!\-       # Match special characters
```

The `-` should be escaped, to not be considered as range specifier (like in `[a-z]`).

Reply:
Escaping this I will end wit error `preg_replace_callback(): Unknown modifier '>'`.

Reply:
🤦 Sorry, this was my fault. Used `/>` in the comment. Fixed.